From: Modestas Vainius <modax@debian.org>
Subject: disable INSTALL_RPATH_USE_LINK_PATH where it triggers useless RPATH
 .
 cantor-backend-r: lintian output: 'binary-or-shlib-defines-rpath ./usr/bin/cantor_rserver /usr/lib64/R/lib', automatically rejected package.
 cantor-backend-r: lintian output: 'binary-or-shlib-defines-rpath ./usr/lib/kde4/cantor_rbackend.so /usr/lib64/R/lib', automatically rejected package.
Forwarded: not-needed
Origin: vendor
Last-Update: 2010-07-27
--- a/src/backends/R/rserver/CMakeLists.txt
+++ b/src/backends/R/rserver/CMakeLists.txt
@@ -17,6 +17,7 @@ add_custom_target(renvvars COMMAND ${R_E
 add_custom_target(rautoloads COMMAND ${R_EXECUTABLE} --slave --file=${CMAKE_CURRENT_SOURCE_DIR}/tools/autoloads.r > ${CMAKE_CURRENT_BINARY_DIR}/rautoloads.h )
 
 kde4_add_executable( cantor_rserver ${RServer_SRCS} )
+set_target_properties( cantor_rserver PROPERTIES INSTALL_RPATH_USE_LINK_PATH false)
 target_link_libraries( cantor_rserver ${KDE4_KDEUI_LIBS} cantorlibs ${R_LIBRARIES} ${KDE4_KIO_LIBS})
 add_dependencies(cantor_rserver renvvars rautoloads)
 
--- a/src/backends/R/CMakeLists.txt
+++ b/src/backends/R/CMakeLists.txt
@@ -21,6 +21,7 @@ QT4_ADD_DBUS_INTERFACE(RBackend_SRCS ${n
 kde4_add_ui_files(RBackend_SRCS settings.ui)
 
 kde4_add_plugin( cantor_rbackend ${RBackend_SRCS} )
+set_target_properties( cantor_rbackend PROPERTIES INSTALL_RPATH_USE_LINK_PATH false)
 target_link_libraries( cantor_rbackend ${KDE4_KDEUI_LIBS} ${KDE4_KIO_LIBS} cantorlibs ${R_USED_LIBS})
 
 install( FILES rbackend.desktop  DESTINATION ${SERVICES_INSTALL_DIR}/cantor)
